scripts/run_interactive_fuzz.sh

```diff
@@ -1,10 +1,10 @@
 # TODO Run these in the background, assuming these will handle how to notify or log errors
 
 echo "HYPERDRIVE BALANCE"
-python lib/agent0/agent0/interactive_fuzz/fuzz_hyperdrive_balance.py;
+python src/agent0/core/interactive_fuzz/fuzz_hyperdrive_balance.py;
```

Comment:
I actually don't think this shell script was ever being used, we have a python script that runs interactive fuzz tests

Reply:
ok I'll delete it